include/srdfdom/model.h

```diff
@@ -279,6 +314,11 @@ class Model
   std::vector<CollisionPair> enabled_collision_pairs_;
   std::vector<CollisionPair> disabled_collision_pairs_;
   std::vector<PassiveJoint> passive_joints_;
+  std::map<std::string, std::vector<JointProperty>> joint_properties_;
```

Comment:
Hm. My actual comment was lost. Here it is again: 
I think this is the wrong data structure to store properties:
- First, the joint name is stored redundantly both, in this map and in the JointProperty struct.
- Second, the vector should actually be a map too (property name -> property value).
  It doesn't make sense to have multiple properties with same name but different value.